packages/steiger/src/features/choose-root-folder/choose-from-similar.ts

```diff
@@ -0,0 +1,120 @@
+import { readdir } from 'node:fs/promises'
+import { parse, relative, sep, join } from 'node:path'
+import pc from 'picocolors'
+import { isGitIgnored } from 'globby'
+
+import { distance } from 'fastest-levenshtein'
+import { isCancel, outro, select, confirm } from '@clack/prompts'
+import { formatCommand } from './format-command'
+import { ExitException } from './exit-exception'
+
+/** The maximum Levenshtein distance between the input and the reference for the input to be considered a typo. */
+const typoThreshold = 5
+const isIgnored = await isGitIgnored()
```

Comment:
Yes, but we use this function a bit differently in `packages/steiger/src/features/transfer-fs-to-vfs.ts` (by finding the git root manually and passing it to `isGitIgnored`). Should we separate this usage into a separate reusable function to make sure we ignore the same files everywhere and avoid some conflicts and weird situations?

Reply:
Good catch. I used the `empathic` library to search for the Git root, so now there's not that much code and I don't think we need to bother with reusability